live-examples/js-examples/expressions/expressions-newtarget.js

```diff
@@ -1,10 +1,10 @@
 function Foo() {
-  if (!new.target) { throw 'Foo() must be called with new'; }
+  if (!new.target) { throw 'TypeError: calling Foo constructor without new is invalid'; }
```

Comment:
Oops, this should really be: `throw new TypeError ("calling Foo constructor without new is invalid")`

But when we one day enable ESLint, this would be caught by `no-throw-literals` anyway.